packages/web3-eth/src/types.ts

```diff
@@ -0,0 +1,194 @@
+import { AccessList, Log, TransactionHash, TransactionInfo, Uncles } from 'web3-common';
+import {
+	Address,
+	HexString,
+	Numbers,
+	ValidTypes,
+	ValidReturnTypes,
+	HexString32Bytes,
+	HexString256Bytes,
+	HexStringBytes,
+	Uint,
+} from 'web3-utils';
+
+export enum ChainNames {
+	MAINNET = 'mainnet',
+	GOERLI = 'goerli',
+	KOVAN = 'kovan',
+	RINKEBY = 'rinkeby',
+	ROPSTEN = 'ropsten',
+}
+
+export enum HardForks {
+	CHAIN_START = 'chainstart',
+	HOMESTEAD = 'homestead',
+	DAO = 'dao',
+	TANGERINE_WHISTLE = 'tangerineWhistle',
+	SPURIOUS_DRAGON = 'spuriousDragon',
+	BYZANTIUM = 'byzantium',
+	CONSTANTINOPLE = 'constantinople',
+	PETERSBURG = 'petersburg',
+	ISTANBUL = 'istanbul',
```

Comment:
[Muir Glacier ](https://ethereum.org/en/history/#muir-glacier)

```suggestion
	ISTANBUL = 'istanbul',
	MUIRGLACIER = muirGlacier,
```

Reply:
Created an [issue](https://github.com/ChainSafe/web3.js/issues/4633) for this, and added it to [Web3Eth TODO list](https://github.com/ChainSafe/web3.js/issues/4608)